man/man8/zfs.8

```diff
@@ -2453,6 +2453,11 @@ original form.
 Displays a help message.
 .It Xo
 .Nm
+.Fl V -version
+.Xc
+Displays the ZFS version information.
```

Comment:
Why are we doing this as a flag, rather than a new subcommand?  You can't do `zfs --version create ...` or any other subcommands, can you?

Reply:
Because most that use GNU (and other) software use the de facto standard method of:
$ program --version
(that is always the first thing I usually try)

just like:
$ program --help

Reply:
That makes sense.  I think we should consider making a `zfs version` subcommand, and have the `--version`/`-V` be a shortcut for running `zfs version`.

Reply:
done